policyengine_us/parameters/gov/irs/deductions/standard/aged_or_blind/amount.yaml

```diff
@@ -62,15 +64,16 @@ HEAD_OF_HOUSEHOLD:
   2024-01-01: 1_950
   2025-01-01: 2_000
   # CBO forecast.
-  2025-02-01: 2_050
-  2025-03-01: 2_100
-  2025-04-01: 2_150
-  2025-05-01: 2_200
-  2025-06-01: 2_200
-  2025-07-01: 2_250
-  2025-08-01: 2_300
-  2025-09-01: 2_350
-  2025-10-01: 2_400
+  2026-01-01: 2_050
```

Comment:
was the previous data monthly? Confused about the change

Reply:
I messed this up last year! 😞 https://github.com/PolicyEngine/policyengine-us/pull/4021